hack/get-server.sh

```diff
@@ -108,5 +108,17 @@ if [[ ! :"$PATH:" == *":$DESTINATION:"* ]]; then
   echo "         Edit your shell configuration file (e.g., ~/.bashrc or ~/.zshrc)"
   echo "         Add the following line:"
   echo "             export PATH=\$PATH:$DESTINATION"
-  echo "         Source the configuration file to apply the changes (e.g., source ~/.bashrc)"
+  echo "         Source the configuration file to apply the changes (e.g., source ~/.bashrc) and run the Daytona Server:"
+  echo "             daytona server"
+  else 
+    # Prompt the user for running the Daytona Server
+    echo -e "\nDo you want to run 'daytona server'? (Y/n) "
+    read choice
```

Comment:
![Screenshot 2024-03-18 at 10 42 57](https://github.com/daytonaio/daytona/assets/26512078/9867b0e5-0e08-47af-a754-e48760acc91f)

The choice read doesn't work. It just exists with no action taken for me (Mac).